test/torch/test_hook.py

```diff
@@ -293,3 +293,15 @@ def initHidden(self):
         # so we better check it beforehand
         assert param.grad.data is not None
         param.data.add_(-learning_rate, param.grad.data)
+
+
+def test_remote_gradient_clipping(workers):
```

Comment:
Can you add a test to make sure that gradient clipping works locally as well

Reply:
Added! And modified the method to support the modification of local tensors with gradient clipping as well.